docs/documentation/release_notes/topics/25_0_0.adoc

```diff
@@ -34,6 +34,31 @@ New options are activated by default, so {project_name} will not recognize the o
 
 For information on how to migrate, see the link:{upgradingguide_link}[{upgradingguide_name}].
 
+= Persistent user sessions
```

Comment:
Would it make sense to add an information that existing online sessions (prior to the upgrade) will not survive the upgrade to 25?

Reply:
The migration might be possible when external ISPN is used, however, it is impossible when only embedded ISPN is used which is I assume the most common setup. What I am afraid of here is that people will expect they won't lose sessions when they enable this feature for their existing setups which is not true.

Reply:
OK, then please add a sentence that describes the current functionality. If we merge the other issue, we can revise it.

Reply:
Added